SmartEVSE-3/src/evse.cpp

```diff
@@ -2914,10 +2920,15 @@ void StartwebServer(void) {
         
         doc["home_battery"]["current"] = homeBatteryCurrent;
         doc["home_battery"]["last_update"] = homeBatteryLastUpdate;
-        
-        doc["ev_meter"]["EVMeter_import_active_energy"] = round(EnergyEV / 100)/10; //in kWh, precision 1 decimal
-        doc["mains_meter"]["MainsMeter_import_active_energy"] = round(Mains_import_active_energy / 100)/10; //in kWh, precision 1 decimal
-        doc["mains_meter"]["MainsMeter_export_active_energy"] = round(Mains_export_active_energy / 100)/10; //in kWh, precision 1 decimal
+
+        doc["ev_meter"]["description"] = EMConfig[EVMeter].Desc;
+        doc["ev_meter"]["address"] = EVMeterAddress;
+        doc["ev_meter"]["power"] = round(PowerMeasured / 100)/10; //in kWh, precision 1 decimal
+        doc["ev_meter"]["total_kwh"] = round(EnergyEV / 100)/10; //in kWh, precision 1 decimal
```

Comment:
To be consistent with the mains_meter items, I would prefer ["ev_meter"]["import_active_energy"] for this item, because that is what is actually shown here; it is the "active" part of the energy (so it ignores the "reactive" part, which you don't pay for so irrelevant for most people), and also it is only the imported (used) part. Now usually on the EV_meter there is no energy source connected (like solar panels), so usually export_active_energy=0, and that means that it is indeed "total kWh". 

But in the future a firmware update is making e.g. my Enyaq bidirectional (so it _can_ deliver energy through the EV_meter), this difference _will_ become relevant.
 
So my request would be to change this!

Reply:
That is fine for me.